docs/document/content/manual/sharding-jdbc/configuration/config-java.en.md

```diff
@@ -290,12 +290,12 @@ The implementation class of `ShardingStrategyConfiguration`, used to configure n
 | *Name* | *DataType* | *Description*                                                |
 | ------ | ---------- | ------------------------------------------------------------ |
 | column | String     | Column name of key generator                                 |
-| type   | String     | Type of key generator, use user-defined ones or built-in ones, e.g. SNOWFLAKE, UUID |
-| props  | Properties | The Property configuration of key generators                 |
+| type   | String     | Type of key generate algorithm, use user-defined ones or built-in ones, e.g. SNOWFLAKE, UUID |
```

Comment:
KeyGeneratorConfigration contains KeyGenerateAlgorithm now, should move the modification to yaml and namespace config